code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -660,15 +671,28 @@ GLOBAL_VAR(medibot_unique_id_gen)
 						speak(message,radio_channel)
 						playsound(src, messagevoice[message], 50)
 						COOLDOWN_START(src, declare_cooldown, 10 SECONDS)
-			if(MEDIBOT_TREAT_SUCK)
+					C.visible_message("<span class='danger'>[src] injects [patient] with its syringe!</span>", \
+					"<span class='userdanger'>[src] injects you with its syringe!</span>")
+			if(MEDBOT_TREAT_BANDAGE)
+				var/mob/living/carbon/human/H = C
+				if(!H.bleed_rate)
+					to_chat(src, "<span class='warning'>[H] isn't bleeding!</span>")
+					return
+				if(H.bleedsuppress) //so you can't stack bleed suppression
+					to_chat(src, "<span class='warning'>[H]'s bleeding is already bandaged!</span>")
+					return
```

Comment:
When bleeding is merged, this will have to be updated to use is_bleeding (which checks if the user has bleeding that isn't bandaged) and is_bandaged (checks if the user has bleeding that is bandaged)